src/main.rs

```diff
@@ -1,3 +1,101 @@
-fn main() {
-    println!("Hello, world!");
+pub mod model;
+pub mod view;
+
+use std::sync::{Arc, RwLock};
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::thread;
+use std::time::Duration;
+use macroquad::prelude::*;
+
+use model::game_of_life::GameOfLife;
+use view::screen::Screen;
+
+
+const INITIAL_AREA: (i32, i32, i32, i32) = (-20, -20, 20, 20);
```

Comment:
Maybe this could abstracted away as a struct with meaningful names (i.e. each coordinate name) since looking at this I can't figure out which coordinate is wich.